lib/mediators/followups/notifier.rb

```diff
@@ -24,14 +23,7 @@ def update_users
 
     def notify_users
       notifications.each do |note|
-        user = note.user
-        emailer = Telex::Emailer.new(
-          email: user.email,
-          in_reply_to: note.id,
-          subject: message.title,
-          body: followup.body
-        )
-        emailer.deliver!
+        Jobs::SendNotificationEmail.perform_async(note.id, followup.id)
```

Comment:
I wonder if we could hide the details of Sidekiq (`::perform_async`) so that consumers of the `SendNotificationEmail` object aren't coupled to them? I'm thinking a facade method, which delegates to Sidekiq's machinery, while providing an interface geared toward our use case. Something like this, maybe?

```suggestion
        Jobs::SendNotificationEmail.call(note: note, followup: followup)
```

With a boring implementation like:

```ruby
module Jobs
  class SendNotificationEmail
    include Sidekiq::Worker

    def self.call(note:, followup:)
      perform_async(note.id, followup.id)
    end

    # ...
  end
end
```

Reply:
This is just how the other jobs work -- 

```
lib/mediators/messages/creator.rb
26:        Jobs::MessagePlex.perform_async(msg.id)

lib/mediators/followups/creator.rb
9:      Jobs::NotifyFollowup.perform_async(@followup.id)

lib/clock.rb
14:    Jobs::Cleanup.perform_async
```

Reply:
Yep, that's fair. Like I said, not a block, but _maybe_ a small design improvement we'd like to make while we're in this bit of code. [I'm like a 3/10 on this one](https://capwatkins.com/blog/the-sliding-scale-of-giving-a-fuck), so your call.